gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 18:09:37 CET 2017
+#Sat Apr 29 11:24:25 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
> I'll dig through history to see if I can find a commit where it switched from bin to all.

I finally got around to doing this...  The `all` version was first used in d17a793.  The message from that commit states:

> use the all version for IDE autocomplete support

I don't have a Groovy plugin installed in my IDE, but I assume that if I did, the `all` version would allow the IDE to provide context assistance for the Gradle API.  Sounds like a good reason to use `all` instead of `bin`. :smiley: